app/views/post/index.html.erb

```diff
@@ -0,0 +1,20 @@
+<% if @posts.blank? %>
+  <h4>There are no posts currently for this user</h4>
+<% else %>
+  <% @posts.each.with_index(1) do|post, index| %>
+    <div id = 'posts'>
+      <div class="card text-center">
+        <div class="card-header">
+          <%= post.title %>
+          <p class="card-text"><small class="text-muted">Comments: <%= post.comments_counter %></small></p>
+          <p class="card-text"><small class="text-muted">Likes: <%= post.likes_counter %></small></p>
+        </div>
+        <div class="card-body">
+          <p class="card-text"><%= post.text %></p>
+          <%= link_to "See Post", user_post_path(post.author, post), class: "btn btn-primary" %>
+        </div>
+      </div>
+    <% end %>
+  </div>
```

Comment:
- [ ] I didn't get why you removed the comment section here, the previous reviewer suggested matching the wireframe but you deleted that code so, please add a comment section with the recent comments belonging to the user's post and try to match the layout with the wireframe.